ci/generate-workflows.py

```diff
@@ -43,7 +47,7 @@ def render(self, f, env):
                 if self.shell == "bash":
                     run = f"export {k}={v}\n{run}\n"
 
-        f.write(f"      run: {yv(run)}\n")
+        f.write(f"{indent}  run: {yv(run, 5)}\n")
```

Comment:
Good point. 5 looks magical. It should actually be `depth + 2` here, because we're writing the run statements of a (potential) multiline block:

```
- name:     <-- depth "0"
  shell:    <-- depth +1  
  run: |    <-- depth +1
    line1   <-- depth +2
    line2   <-- depth +2